packages/flutter/lib/src/material/radio_list_tile.dart

```diff
@@ -419,11 +435,6 @@ class RadioListTile<T> extends StatelessWidget {
   /// When null, `EdgeInsets.symmetric(horizontal: 16.0)` is used.
   final EdgeInsetsGeometry? contentPadding;
 
-  /// Whether this radio button is checked.
-  ///
-  /// To control this value, set [value] and [groupValue] appropriately.
-  bool get checked => value == groupValue;
```

Comment:
I wonder if this might cause a little breakage. Probably something to mention in the migration guide for this change explaining some of the motivations here re: accessibility

Reply:
will include this in migration guide